roles/bond-config/tasks/main.yml

```diff
@@ -6,7 +6,7 @@
 ##########################################################################################################
 
 - include: redhat.yml
-  when: ansible_distribution == 'CentOS' or ansible_distribution == 'Red Hat Enterprise Linux'
+  when: ansible_distribution == 'CentOS' or ansible_distribution == 'RedHat'
```

Comment:
This should be `when: ansible_os_family == "RedHat"`

Example https://github.com/platform9/express/blob/master/roles/neutron-prerequisites/tasks/main.yml